File: HHB/BJ/No_1316.java
package HHB.BJ;
import java.io.*;

public class No_1316 {
    public static void main(String args[]) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int n = Integer.parseInt(br.readLine()); //입력 받을 그룹 단어의 개수
        int count = n; //그룹단어의 개수

        for(int i=0; i<n; i++){
            boolean[] chk = new boolean[26]; //false로 초기화되어있음
            String str = br.readLine();

            for(int j=0; j<str.length(); j++){
                if(j==0){
                    chk[str.charAt(j)-'a'] = true;
                }
                else{
                    if(str.charAt(j-1)==(str.charAt(j))){
                        break;
                    }
                    else{
                        if(!chk[str.charAt(j)-'a']){
                            chk[str.charAt(j)-'a'] = true;
                        }
                        else{
                            count--;
                            break;
                        }
                    }
                }
            }
        }

        System.out.print(count);

        br.close();
    }
}

File: HHB/BJ/No_1316.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});

let input = [];

rl.on("line", (line) => {
    input.push(line);
}).on("close", () => {
    let n = Number(input[0]); //입력 받을 그룹 단어의 개수
    let count = n; //그룹단어의 개수

    for(let i=0; i<n; i++){
        let chk = [];
        let str = input[i];

        for(let j=0; j<str.length; j++) {
            if(!chk.includes(str[j])) {
                chk.push(str[j]);
            }
            else {
                if(str[j] === str[j-1]){
                    break;
                }
                else{
                    count--;
                    break;
                }
            }
        }

    }
    console.log(count);

    process.exit();
});